test/setup/main.tf

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  project_name   = "cft-sap"
+  subfolder_name = "${local.project_name}-${random_id.folder_rand.hex}"
+}
+
+resource "random_id" "folder_rand" {
+  byte_length = 2
+}
+
+resource "google_folder" "ci_event_func_subfolder" {
```

Comment:
We shouldn't need a new folder at all?

Reply:
You are right! Sorry, I should have reviewed this part in more detail. I'm removing this.

Reply:
I removed these unnecessary folders (and a couple other resources).